gen3workflow/routes/storage.py

```diff
@@ -15,10 +15,14 @@ async def get_storage_info(request: Request, auth=Depends(Auth)) -> dict:
     user_id = token_claims.get("sub")
     logger.info(f"User '{user_id}' getting their own storage info")
     bucket_name, bucket_prefix, bucket_region = aws_utils.create_user_bucket(user_id)
+    _, kms_key_arn = aws_utils.get_existing_kms_key_for_bucket(bucket_name)
     return {
         "bucket": bucket_name,
         "workdir": f"s3://{bucket_name}/{bucket_prefix}",
         "region": bucket_region,
+        "kms_key_arn": (
+            kms_key_arn if config["KMS_ENCRYPTION_ENABLED"] and kms_key_arn else None
+        ),
     }
```

Comment:
Instead of calling `get_existing_kms_key_for_bucket` again and making additional calls to AWS, `create_user_bucket` could just return `kms_key_arn = <ARN or None>`
```suggestion
    bucket_name, bucket_prefix, bucket_region, kms_key_arn = aws_utils.create_user_bucket(user_id)
    return {
        "bucket": bucket_name,
        "workdir": f"s3://{bucket_name}/{bucket_prefix}",
        "region": bucket_region,
        "kms_key_arn": kms_key_arn,
    }
```

Reply:
done in https://github.com/uc-cdis/gen3-workflow/pull/93